scripts/launch/demo.sh

```diff
@@ -85,6 +85,14 @@ echo "Using system start time "$system_start
 
 echo "Number of panes: $panesCnt"
 
+function prefix_args {
```

Comment:
What does this function do? I mean, I can read the code and understand each step, but... how would one even describe it? :thinking:

Reply:
> understand each step

You are smart, I have no idea what's going on here even if I read the code :/

Reply:
¯\_(ツ)_/¯ my task was to add logging, I am not sure who wrote the function, I don't mess too much with bash scripts

Reply:
Right. I won't even try to understand bash code, will only check that nothing is broken. And if something is broken due to bash code, I will ask George to fix it, because it's he who wrote it.